### PR TITLE
Cleaning CSS, updating packages

### PR DESCRIPTION
**Pull request by ravindk89**
# Description

General fixups to the CSS to make things look a little cleaner, and to fix some random broken stuff.

See #minio-docs for staging URL.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 sphinx == 3.5.0
```
Comment (rushenn) [inline requirements.txt:1]: @ravindk89 Looks like extension `myst_parser` is missing from `requirements.txt`, leading to an error while building the HTML.

Comment (rushenn) [reply requirements.txt:1]:
```
Extension error:
Could not import extension myst_parser (exception: No module named 'myst_parser')
```

Comment (ravindk89) [reply requirements.txt:1]: I cleaned out my local venv, fixed requirements.txt, and tried again - error appears to be cleared out.

Comment (ravindk89) [reply requirements.txt:1]: @rushenn LMK if you still run into trouble.

**Review by rushenn — APPROVED** [review]
LGTM

**Pull request merged**